src/app.js

```diff
@@ -104,12 +111,14 @@ export const start = args => {
 
     doFetchBranches()
       .then(() => {
+        branchTable.clearItems();
+
         refreshTable(currentRemote);
       })
       .catch(error => {
         if (error !== 'SIGTERM') {
           screen.destroy();
-
+  
```

Comment:
We have a trailing space here, the latest update to development included a 'format' script. `npm run format` should call prettier on all the files. You can commit the prettier changes together if it updates anything else.